xml/System.Runtime.InteropServices/TypeMapAssociationAttribute`1.xml

```diff
@@ -32,7 +32,40 @@
   <Docs>
     <typeparam name="TTypeMapGroup">The type of the type map group.</typeparam>
     <summary>Represents an attribute that creates a type association between a type and its proxy.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Types used in a managed-to-unmanaged interop operation can use `TypeMapAssociationAttribute` to define a conditional link between the source and proxy type. If the source is kept, so is the proxy type. If trimming observes an explicit allocation of the source type, the entry is inserted into the map.
+
+When an application is trimmed, an entry in the Proxy Type Map is included when the "source type" is referenced in one of the following ways in reachable code:
```

Comment:
All of this below is incorrect, it only applies to the type map, not the proxy map.